streams/readable-streams/bad-underlying-sources.js

```diff
@@ -82,9 +80,11 @@ promise_test(t => {
 
   return Promise.all([
     reader.read().then(r => {
-      assert_object_equals(r, { value: 'a', done: false }, 'the chunk read should be correct');
+      assert_object_equals(r, { value: 'a', done: false }, 'the first chunk read should be correct');
     }),
-    promise_rejects(t, theError, reader.closed)
+    reader.read().then(r => {
+      assert_object_equals(r, { value: 'a', done: false }, 'the second chunk read should be correct');
+    })
   ]);
 
 }, 'Underlying source pull: throwing getter (second pull)');
```

Comment:
This test might benefit from a comment saying that the test is that it's only gotten once. Indeed, asserting that counter === 1 after the last assert_object_equals might be good.

Reply:
Done.